backend/Application/Application.csproj

```diff
@@ -58,7 +59,7 @@
             </None>
     </ItemGroup>
 
-    <Target Name="BuildRustProject" BeforeTargets="Build">
+    <Target Name="BuildRustProject" BeforeTargets="Build" Condition="$(RunRustBuild)">
         <Exec Command="cargo build --manifest-path=../rust-bindings/Cargo.toml --release" />
     </Target>    
```

Comment:
Discussed offline, included here for reference: It may be that this command rebuilds the target after if already copied the existing version.

This doesn't matter in the docker build because you build the bindings manually before the dotnet project, but it could be a source of confusion during development.